plugins/native/configs/config.d/10_native.json

```diff
@@ -62,6 +62,26 @@
       "translucency-support": false
     }
   },
+  {
+    "match": ".*(spz)",
```

Comment:
technicaly you could share almost everything with .splat and only specify the camera position for each individual format

Reply:
not adressed ?